packages/react-button/src/components/Button/Button.stories.tsx

```diff
@@ -62,32 +62,34 @@ CircularButton.parameters = {
   },
 };
 
-export const ButtonSize = () => (
-  <>
-    <div>
-      <h4>small</h4>
-      <Button size="small">Text</Button>
-      <Button size="small" icon={<CalendarMonth24Regular />}>
-        Text
-      </Button>
-      <Button size="small" icon={<CalendarMonth24Regular />} />
-    </div>
-    <div>
-      <h4>medium</h4>
-      <Button>Text</Button>
-      <Button icon={<CalendarMonth24Regular />}>Text</Button>
-      <Button icon={<CalendarMonth24Regular />} />
-    </div>
-    <div>
-      <h4>large</h4>
-      <Button size="large">Text</Button>
-      <Button size="large" icon={<CalendarMonth24Regular />}>
-        Text
-      </Button>
-      <Button size="large" icon={<CalendarMonth24Regular />} />
-    </div>
-  </>
-);
+export const ButtonSize = () => {
+  return (
+    <>
+      <div style={{ display: 'flex', flexWrap: 'wrap', gap: '1em' }}>
```

Comment:
nit: some repeated styles and why is the gap 1em **and** 0.5em ?

Reply:
fixed